lib/avtp_pipeline/openavb_common/avb.c

```diff
@@ -67,30 +63,27 @@ int pci_connect(device_t * igb_dev)
 		snprintf(devpath, IGB_BIND_NAMESZ, "%04x:%02x:%02x.%d",
 				dev->domain, dev->bus, dev->dev, dev->func);
 		err = igb_probe(igb_dev);
-		if (err)
+		if (err) {
 			continue;
-
+		}
 		printf("attaching to %s\n", devpath);
 		err = igb_attach(devpath, igb_dev);
 		if (err) {
-			printf("attach failed! (%s)\n", strerror(err));
+			printf("attach failed! (%s)\n", strerror(err)); /*errno or err ??*/
```

Comment:
Looking at the implementation, I would say that `err` is the desired variable.  Should probably be updated in the other files that use `errno`.

It appears to be setting the return value to the value of `errno` already in the case where the APIs called use it to return their errors, and using standard error codes in other cases.

Do @intel-ethernet or @christopher-s-hall have any comments on this?